tests/legacy-cli/e2e/utils/process.ts

```diff
@@ -299,22 +300,21 @@ export function silentNpm(
       {
         silent: true,
         cwd: (options as { cwd?: string } | undefined)?.cwd,
-        env: extractNpmEnv(),
```

Comment:
None of these are needed anymore because the child process has the correct env passed in already, see `launchTestProcess`.